frontend/src/metabase/actions/components/ActionViz/ActionDashcardSettings.unit.spec.tsx

```diff
@@ -73,9 +103,18 @@ const actionDashcardWithAction = createMockActionDashboardCard({
   id: 3,
   action: actions2[2],
 });
+const actionDashcardWithActionWithHiddenFields = createMockActionDashboardCard({
+  id: 4,
+  action: actions2[3],
```

Comment:
nit: I think this is hard to follow what action it's trying to create. I mean the name `actionDashcardWithActionWithHiddenFields` is pretty self-explanatory, but `actions2` is not clear also the magic index number `3`.


do you think it'd be better if you structure variables a bit differently e.g.
#### This is a pseudo code but to demonstrate an idea
```
const actionWithHiddenFields = ... // whatever you put in `actions2[3]`
```
then we could do

```suggestion
  action: actionWithHiddenFields,
```

As a bonus, if `actions2` should have a meaning on its own maybe we could also name it.
```
const actionsForSomething = [actionWithFoo, actionWithBar, actionWithBaz, actionWithHiddenFields]
```

Reply:
I agree, let me try to come up with some better names

Reply:
Fixed only the main place (for `actionDashcardWithActionWithHiddenFields`), as for other places, actions are similar and they are basic, so I can't come up with some good name, probably array structure works the best in this case